html/tutorial.html

```diff
@@ -1357,7 +1357,7 @@ <h3>Using the Native Fulltext Search</h3>
 <code class="notranslate">KEYS</code> (an array of objects).
 To join a table, use a join as in:
 <code class="notranslate">SELECT T.* FROM FT_SEARCH_DATA('Hello', 0, 0) FT, TEST T
-WHERE FT.TABLE='TEST' AND T.ID=FT.KEYS[0];</code>
+WHERE FT."TABLE"='TEST' AND T.ID=FT.KEYS[1];</code>
```

Comment:
Please, quote `KEYS` too. It isn't a reserved word, but with `DATABASE_TO_LOWER=TRUE` unquote identifier `KEYS` means `"keys"`. This table always has upper-case columns `"SCHEMA"`, `"TABLE"`, `"COLUMNS"`, and `"KEYS"`. By default `"keys"` is not equal to `"KEYS"`.